libraries/ESP8266WiFi/src/ESP8266WiFiSTA.cpp

```diff
@@ -86,6 +86,7 @@ static bool sta_config_equal(const station_config& lhs, const station_config& rh
 // -----------------------------------------------------------------------------------------------------------------------
 
 bool ESP8266WiFiSTAClass::_useStaticIp = false;
+bool ESP8266WiFiSTAClass::_useInsecureWEP = false;

```

Comment:
Is this here on purpose, or did it slip in? I'm ok with what I see of the code, but given the nature of this functionality I suggest splitting it into its own PR.

Reply:
All points above are addressed your way.
About WEP, it is a requirement to set this value in sdk structures for initialization.